.eslintrc.cjs

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
+module.exports = {
+	extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+	plugins: ['@typescript-eslint'],
+	ignorePatterns: ['/*', '!/src', '*.test.ts'],
+	rules: {
+		'no-unused-vars': ['error', { vars: 'all', args: 'none' }],
+		'@typescript-eslint/ban-ts-comment': ['error', { 'ts-ignore': 'allow-with-description' }],
+	},
+	parser: '@typescript-eslint/parser',
+	parserOptions: {
+		project: true,
+		tsconfigRootDir: __dirname
+	},
+	root: true
+};
```

Comment:
Could we make this re-usable for plugins somehow?

Reply:
We should! I think eslint configs can live in repos as well.

Reply:
I'll take care of it.